### PR TITLE
nep-13-add-push-notification: firebase implemention initialized

### PR DESCRIPTION
**Pull request by shrijan-siris**
- [x] Include Jira Ticket No. in Title
- [ ] Write Unit Tests
- [ ] Add screenshots whenever possible
- [ ] Make sure Github Action build succeed
- [ ] Address Automated Code Reviews from Codacy, LGTM and Deepscan
- [x] Add Reviewers

### CODE REVIEW

File: android/app/google-services.json [hunk 1]
```diff
@@ -0,0 +1,40 @@
+{
```
Comment (syuraj) [inline android/app/google-services.json]: @shrijan-siris there is no secret information in this file right? also, can you generate this file from the firebase app i created

Comment (shrijan-siris) [reply android/app/google-services.json]: yes there is no secret information , and i will replace this one for sure

**Comment by syuraj** [comment]
Will it also work for IPhone app?

**Comment by shrijan-siris** [comment]
> Will it also work for IPhone app?

yes this will works for both

**Pull request merged**